app/routes/archive/components/ItemArchivePanel.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { PropTypes } from "react"
+import classNames from "classnames"
+import ItemPanel from "../../shared/components/ItemPanel"
+
+const containerStyles = {
+  marginTop: "10px"
+}
+
+const ItemArchivePanel = ({
+  id,
+  username,
+  displayName,
+  avatarUrl,
+  repoUrl,
+  progress
+}) => {
+  let viewButton, barColorClass
+
+  if (progress === 100) {
+    barColorClass = "progress-bar-success"
+    viewButton = (
+      <div className="col-sm-4">
+        <button className="btn btn-xs btn-primary">
+          <i className="fa fa-folder-open" aria-hidden="true"></i> View
+        </button>
+      </div>
+    )
+  } else {
+    barColorClass = "progress-bar-info"
+    viewButton = (
+      <p> Archiving.. </p>
+    )
+  }
+
+  const progressBarClasses = classNames("progress-bar", barColorClass)
+
+  return (
+    <ItemPanel
+      imagePath={avatarUrl}
+      title={username}
```

Comment:
My two cents: what do you think about prepending an `@` sign before the `username`?